### DISCUSSION TITLE
[Feature Request] Remember active tab

### DISCUSSION THREAD

**Discussion by Cerothen** [kinghat/tabbed-card#81] [Feature Requests]
Great card its really been helping tie some of the views together nicely.

One thing that would be nice if the location.hash property was set (or something) so that the selected tab was remembered. Sometimes the app or page refreshes (when coming back) and it doesnt remember which tab was currently being utilized.

Thanks for the quality work!

**Comment by kinghat** [comment]
are you saying that when you select x tab that it will just randomly reset to the default tab? or that the browsers page is reloaded or killed for being inactive and you come back to it, its gone back to the default tab?

**Comment by Cerothen** [comment]
When the browser is reloaded it always selects the first tab, home assistant selects the Lovelace tab but then the tab card always goes back to the original

**Comment by kinghat** [comment]
you can set the default tab, but it will always be that tab when reloaded. as to having the card remember your last tab position i cant say that i would implement that. it would be easy enough to continuously set the default tab to whatever tab you navigate to. ill keep it in mind when expanding out features.

im going to move this to the feature request discussion area.
